go/proto/assetmanagerd/v1/asset.proto

```
@@ -1,15 +1,15 @@
 syntax = "proto3";
 
-package deploy.assetmanagerd.v1;
+package assetmanagerd.v1;
```

Buf (1.55.1)

Files with package "assetmanagerd.v1" must be within a directory "assetmanagerd/v1" relative to root but were in directory "go/proto/assetmanagerd/v1".

(PACKAGE_DIRECTORY_MATCH)